components/brave_wallet/browser/BUILD.gn

```diff
@@ -18,6 +16,8 @@ source_set("browser") {
 
     deps = [
         "//base",
+        "//brave/common:common",
```

Comment:
these are illegals dep that should be captured by `gn check`. We should remove them and fix the corresponding includes

Reply:
👍 I posted this issue to fix https://github.com/brave/brave-browser/issues/11598